commands/theme/marketplace-validate.ts

```diff
@@ -39,28 +51,30 @@ exports.handler = async options => {
     assetType,
     path
   );
-  await pollForValidationFinish(derivedAccountId, validationId);
+  await pollForValidationFinish(derivedAccountId, validationId.toString());
```

Comment:
It's probably more likely that `validationId` arg for this function was improperly typed - can switch it to be a number